docs/release-notes.md

```diff
@@ -19,6 +19,7 @@ nav_order: 9
 
 - Document that `hash` fields describe decompressed data
 - Correctly document Tang `advertisement` field as optional
+- Fix relabling mountpoints which already exist
```

Comment:
hmm that verbiage sounds like we are allowing the relabeling of mountpoints which already exist. 

WDYT about
"Correctly avoid relabeling a mount point that exists"

Reply:
SGTM!